### PR TITLE
Invoices.updateBooked

### PR DESCRIPTION
**Pull request by Jdww**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -3383,6 +3384,34 @@ Update a draft invoice. Booked invoices cannot be updated.
 
 + Response 204
 
+### invoices.updateBooked [POST /invoices.update]
+
+Update a booked invoice. Only available if "Booked invoices can't be edited" is disabled in the settings.
```
Comment (tiagobastos) [inline apiary.apib]: Do you think we should make reference to the setting instead of a hard-coded label that can change?

**Review by joaoinacio — DISMISSED** [review]

**Review by tiagobastos — DISMISSED** [review]

**Review by joaoinacio — DISMISSED** [review]

**Comment by joaoinacio** [comment]
✅

**Review by joaoinacio — DISMISSED** [review]

**Review by mrtus — DISMISSED** [review]

**Review by tiagobastos — DISMISSED** [review]

**Review by adognameddog — DISMISSED** [review]

**Review by joaoinacio — DISMISSED** [review]

**Review by tiagobastos — DISMISSED** [review]

**Review by joaoinacio — DISMISSED** [review]

File: apiary.apib [hunk 2]
```diff
@@ -428,13 +428,14 @@ We list all backwards-compatible additions here. These are currently available i
...
 - We added `late_fees` to `invoices.info` and `invoices.list`.
+- We added the `invoices.updateBooked` endpoint.
 
 ### September 2023
 - We added the `meetings.list` and `meetings.info` endpoints.
 - We added `time_estimated` and `amount_unbilled` to `projects-v2/projects.list`.
 - We added sorting on `amount_billed`, `amount_paid`, `amount_unbilled`, `external_budget_spent`, `external_budget`, `internal_budget` and `time_estimated` to `projects-v2/projects.list`.
 
-### August 2023
+#### August 2023
```
Comment (adognameddog) [inline apiary.apib:438]: Q: why do we make this a H4 while the other months are a H3?

Comment (joaoinacio) [reply apiary.apib:438]: > Q: why do we make this a H4 while the other months are a H3?

was resolved on master, this brought it back with a merge with master. thanks for noticing

Comment (Jdww) [reply apiary.apib:438]: The other months are actually H4, it's only the 2 most recent ones that are H3. 🤷 fixing it.

**Review by tiagobastos — APPROVED** [review]

**Review by joaoinacio — APPROVED** [review]

**Review by tiagobastos — APPROVED** [review]
✅

**Review by gdegirmenci — APPROVED** [review]

**Pull request merged**